SPECS/espeak-ng/espeak-ng.spec

```diff
@@ -1,16 +1,17 @@
 Summary:        Compact text-to-speech synthesizer
 Name:           espeak-ng
-Version:        1.50
-Release:        3%{?dist}
+Version:        1.51.1
+Release:        1%{?dist}
 # Apache2 license applies only to Android APK code- does not apply here
 # BSD license applies only to Windows code- does not apply here
-License:        GPLv3 AND Unicode
+License:        GPL-3.0-only AND GPL-3.0-or-later AND Unicode-DFS-2016 AND CC-BY-SA-3.0
```

Comment:
I didn't get this change.  Are you saying GPLv3 representation has changed?

Reply:
No, I will update this.